package.json

```diff
@@ -42,6 +43,7 @@
     "path": "^0.12.7",
     "react": "^16.8.2",
     "react-dom": "^16.8.2",
+    "react-google-autocomplete": "^2.6.1",
```

Comment:
Also you can uninstall your installed packages when you are not using them
fx `npm uninstall react-google-autocomplete`